app/views/layouts/application.html.haml

```diff
@@ -19,6 +19,7 @@
       = title
 
     = stylesheet_pack_tag 'common', media: 'all'
+    = stylesheet_pack_tag (current_account&.user&.setting_theme || Setting.default_settings['theme']), media: 'all'
```

Comment:
law of demeter violation. this should probably be a method in ApplicationController instead